app/main.py

```diff
@@ -1,2 +1,49 @@
+class Node:
+    def __init__(self, key: object, value: object) -> None:
+        self.key = key
+        self.value = value
+        self.hash = hash(key)
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = 16, load_factor: float = 0.75) -> None:
```

Comment:
These attrs shouldnt be parametrized - it's part of dict internal structure. Also load factor must be 2/3 - use Fraction for  that